### ISSUE TITLE
Fix commit log for specs with monorepo

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria#2238]
Since we now have a monorepo, the automatic commit log generation will not work (its from the whole history, not history of a file).

**Comment by pkra** [comment]
Does  #2288 address this?

**Comment by daniel-montalvo** [comment]
@pkra 

No. It just deals with removing the manually edited participants files, but tries to harmonize, because there were specs that had different acknowledgements.

**Comment by pkra** [comment]
@jnurthen Is this a duplicate of #1807?

**Comment by daniel-montalvo** [comment]
Closing this after https://github.com/speced/respec/pull/5099 has been merged. Will be available at the next respec release.

**Comment by pkra** [comment]
Awesome - thanks @daniel-montalvo !
